src/email/templates/html.ejs

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Password Reset</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+  </head>
+  <body>
+    <div class="email-container">
+      <h2>Password Reset Request</h2>
+      <p>Hello,</p>
+      <p>
+        We received a request to reset your password. Click the link below to
+        reset your password:
+      </p>
+      <p><a href="<%= resetLink %>">Reset Password</a></p>
+      <% /* The <%= resetLink %> placeholder should be replaced with the actual
+      URL later to the reset password page.*/ %>
```

Comment:
To do a comment in EJS, you need to use `<%# my comment %>` or `<%/* comment */%>`

[https://ejs.co/#docs:~:text=%3C%25%23%20Comment%20tag%2C%20no%20execution%2C%20no%20output](https://ejs.co/#docs:~:text=%3C%25%23%20Comment%20tag%2C%20no%20execution%2C%20no%20output)

Though I think even with that EJS will get a bit confused here about where the starting and ending `<%` are because you have that inner `<%= resetLink %>` inside the comment. So you may want to break that up a bit:

```suggestion
      <%/* The <\% resetLink %\> placeholder should be replaced with the actual
      URL later to the reset password page. */%>
```